microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1184,46 +1184,73 @@ Get information about running WebView2 processes, exiting processes, and failed
 * [CoreWebView2ProcessFailedEventArgs Class](/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedeventargs)
+   * [CoreWebView2ProcessFailedEventArgs.FrameInfosForFailedProcess Property](/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedeventargs.frameinfosforfailedprocess)
+* [CoreWebView2ProcessFailedKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedkind)
+   * [CoreWebView2ProcessFailedKind.GpuProcessExited Enum Value](/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedkind)
+   * [CoreWebView2ProcessFailedKind.UtilityProcessExited Enum Value](/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedkind)
+* [CoreWebView2ProcessFailedReason Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedreason)
 * [CoreWebView2ProcessInfo Class](/dotnet/api/microsoft.web.webview2.core.corewebview2processinfo)
 
-
 ##### [WinRT/C#](#tab/winrtcsharp)
 
 * `CoreWebView2` Class:
    * [CoreWebView2.BrowserProcessId Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#browserprocessid)
+   * [CoreWebView2.Reload Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#reload)
    * [CoreWebView2.ProcessFailed Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#processfailed)
 * [CoreWebView2BrowserProcessExitedEventArgs Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2browserprocessexitedeventargs)
+* `CoreWebView2Controller` Class:
+   * [CoreWebView2Controller.Close Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controller#close)
 * `CoreWebView2Environment` Class:
-   * [CoreWebView2Environment.BrowserProcessExited Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#browserprocessexited)
+   * [CoreWebView2Environment.FailureReportFolderPath Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#failurereportfolderpath)
    * [CoreWebView2Environment.GetProcessInfos Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#getprocessinfos)
+   * [CoreWebView2Environment.BrowserProcessExited Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#browserprocessexited)
    * [CoreWebView2Environment.ProcessInfosChanged Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#processinfoschanged)
 * [CoreWebView2ProcessFailedEventArgs Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2processfailedeventargs)
+   * [CoreWebView2ProcessFailedEventArgs.FrameInfosForFailedProcess Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2processfailedeventargs#frameinfosforfailedprocess)
```

Comment:
@mikehoffms The pattern here seems a bit confusing now. We are listing specific properties of the event args/enum classes, but:
* We're still linking to the class in the parent bullet point, which is inconsistent with the other classes in the list
* We're only linking to only some of the properties/values, but unlisted ones are also related to process management.
* The entire classes we have added are already related to process management.

Do we need to individually list those specific properties/values?

Reply:
CoreWebView2 Class - Keep as-is; don't link to this big/broad class; only link to its specific members.

CoreWebView2BrowserProcessExitedEventArgs Class - Keep as-is; continue to only link to this class, not its members.

CoreWebView2Controller Class  - Keep as-is; don't link to this big/broad class; only link to its Close method.

CoreWebView2Environment Class  - Keep as-is; don't link to this big/broad class; only link to members FailureReportFolderPath Property, GetProcessInfos Method, BrowserProcessExited Event, ProcessInfosChanged Event.

CoreWebView2ProcessFailedEventArgs Class - Continue linking to type.  **Probably remove item here: FrameInfosForFailedProcess Property.**  That property is linked to from article "Handling process-related events in WebView2" in one single-link tab-set.

CoreWebView2ProcessFailedKind Enum - Continue linking to type.  **Probably remove the two member (value) links.**

CoreWebView2ProcessFailedReason Enum - Keep as-is; continue linking to type.

CoreWebView2ProcessInfo Class - Keep as-is; continue linking to type.